cmd/geth/main.go

```diff
@@ -188,6 +188,7 @@ var (
 		utils.AllowUnprotectedTxs,
 		utils.BatchRequestLimit,
 		utils.BatchResponseMaxSize,
+		utils.PreconfirmationForwardingURLFlag,
```

Comment:
I believe this flag is not needed anymore? @cyberhorsey

Reply:
yep can go, i can remove it on my preconfs branch if i havent yet